app/assets/stylesheets/content/_datepicker.sass

```diff
@@ -115,6 +115,7 @@ $dp-shadow-box-opacity: 1
       .ui-datepicker-year
         width: $dp-month-year-width
       select
```

Comment:
@vcostin outside of the realm of third-party components we try to avoid element selectors. I think it's ok to make an exception here though.